src/org/mirah/builtins/string_extensions.mirah

```diff
@@ -39,4 +39,27 @@ class StringExtensions
       end
     end
   end
+  
+  # separate to to_i(), as to_i() (in order to maintain Ruby compatibility) could return a BigInteger,
+  # while to_int() always returns an int.
+  macro def to_int:int
```

Comment:
I think `int('1')` would go down the slippery slope of C++ style converting constructors. I'd rather limit casts to their current use and not expand them, as they are ultimatively a Java feature unknown (and unnecessary) in Ruby.

For the purpose of converting one type into another, ruby has the "to_*" idiom, and I think this idiom completely covers the usages. So the Rubyish solution would be `'1'.to_int`, `'1'.to_Integer` if you like, and I'd prefer that.